### ISSUE TITLE
Warning: License for package Android SDK Build-Tools 25.0.3 not accepted

### ISSUE THREAD

**Issue by nicolasd** [jangrewe/gitlab-ci-android#25]
Gitlab failed to compile since some days:

Checking the license for package Android SDK Build-Tools 25.0.3 in /sdk/licenses
Warning: License for package Android SDK Build-Tools 25.0.3 not accepted.
Checking the license for package Android SDK Platform 25 in /sdk/licenses
Warning: License for package Android SDK Platform 25 not accepted.

Any idea ?

thanks

**Comment by dekzitfz** [comment]
its also happen to me,,, pls fix it

**Comment by Afur** [comment]
I also have same issue

**Comment by vhladiienko** [comment]
This is not related to this particular Docker image, as it happens with me using another image with Android SDK.

**Comment by nicolasd** [comment]
@atomaka i still have the same issue with the new license key

i looked on my computer's licence keys and seems that i have another licence key in the android-sdk-preview-licence:

```
504667f4c0de7af1a06de9f4b1727b84351f2910
84831b9409646a918e30573bab4c9c91346d8abd
```

I tried to push a fork into hub.docker, but the compilation failed each times on 

```
Loading package information...                                                  
Loading local repository...                                                     
[                                       ] 3% Loading local repository...        
[                                       ] 3% Fetch remote repository...         
[                                       ] 3% Fetch remote repository...         
[=                                      ] 3% Fetch remote repository...         
[=                                      ] 3% Fetch remote repository...         
[=                                      ] 3% Fetch remote repository...         
[=                                      ] 3% Fetch remote repository...         
[=                                      ] 3% Fetch remote repository...         
[=                                      ] 3% Fetch remote repository...         
[=                                      ] 3% Fetch remote repository...         
[=                                      ] 3% Fetch remote repository...         
[=                                      ] 3% Fetch remote repository...         
[=                                      ] 3% Fetch remote repository...         
[=                                      ] 3% Fetch remote repository...         
[=                                      ] 4% Fetch remote repository...         
[=                                      ] 4% Fetch remote repository...         
[=                                      ] 4% Fetch remote repository...         
[=                                      ] 4% Fetch remote repository...         
[=                                      ] 4% Fetch remote repository...         
[=                                      ] 4% Fetch remote repository...         
[=                                      ] 4% Fetch remote repository...         
[=                                      ] 4% Fetch remote repository...         
[==                                     ] 5% Fetch remote repository...         
[==                                     ] 6% Fetch remote repository...         
[==                                     ] 6% Computing updates...               
[===                                    ] 8% Computing updates...               
[91mWarning: Failed to find package add-ons;addon-google_apis-google-24
[0m
[===                                    ] 8% Computing updates...               
[===                                    ] 10% Computing updates...              
                                                                                
[===                                    ] 10% Computing updates...              
Removing intermediate container 2e5ad748823b

The command '/bin/sh -c while read -r package; do PACKAGES="${PACKAGES}${package} "; done < /sdk/packages.txt &&     ${ANDROID_HOME}/tools/bin/sdkmanager ${PACKAGES}' returned a non-zero code: 1
```

**Comment by benliger** [comment]
A had the same problem in another docker image: https://github.com/appunite/docker/issues/60

The solution is to change the license : 
Before we had:
> echo -e "\n8933bad161af4178b1185d1a37fbf41ea5269c55" > "${ANDROID_HOME}/licenses/android-sdk-license"

you need to switch to:
> echo -e "\n8933bad161af4178b1185d1a37fbf41ea5269c55\nd56f5187479451eabf01fb78af6dfcb131a6481e" > "${ANDROID_HOME}/licenses/android-sdk-license"

Good luck!

**Comment by jangrewe** [comment]
@nicolasd i've added the additional preview license key to the Dockerfile, but i was able to build the image with the fix from #28 just fine, too (the image uses 26.0.2, though).
I'm happy to add any license keys you guys need, but i'm usually only testing/building the image for the latest stable version.

**Comment by leiliping** [comment]
I also have the same issue
